tests/publiccloud/create_aistack_env.pm

```diff
@@ -159,10 +159,16 @@ sub install_aistack_chart {
     # After reaching max_retries , record the pod details which does not run after reaching max_retries
     my $max_retries = 15;
     my @failed_pods;
+    my @issue_logs_pod;
     my $sleep_interval = 20;
+    my $ollama_pod;
     my @out = split(' ', script_output("kubectl get pods --namespace $namespace -o custom-columns=':metadata.name'"));
     record_info("Pod names", join(" ", @out));
   POD_LOOP: foreach my $pod (@out) {
+
+        if ($pod =~ /^ollama/) {
+            $ollama_pod = $pod;
```

Comment:
If you expect 1 only possible `ollama pod`, then all is ok.

Is there any possiblity that `get pods` would return 2 or more? In that case you'd lost those next, so, be sure that 1 only possible here.